packages/pyright-internal/package.json

```diff
@@ -36,6 +36,7 @@
         "@types/command-line-args": "^5.2.0",
         "@types/fs-extra": "^11.0.1",
         "@types/jest": "^29.5.3",
+        "@types/lodash": "^4.14.202",
```

Comment:
Hmm, I generally try to avoid lodash. It has grown very large over the years, and because it's so intertangled, tree shaking isn't able to remove the unused parts. Please confirm that you're not bloating the bundle size by a large amount by including this. If you are, it would be better to just write your own version of the function(s) you need.

Reply:
This is just the types and it's not included in the actual bundle. It was used in the config.test.ts

Reply:
But I noticed that you're calling `deepCopy` from `lodash` in another part of this PR.

Reply:
In config.test.ts.

Reply:
Ah, got it. So this isn't in the production bundle.